### ISSUE TITLE
Image metrics test broken

### ISSUE THREAD

**Issue by julianpistorius** [cyverse/atmosphere#294]
```
Failure
Traceback (most recent call last):
  File "/opt/dev/atmosphere/api/tests/v2/test_image_metrics.py", line 159, in test_accurate_application_statistics
    self.assertEquals(response.status_code, 200)
AssertionError: 404 != 200
```

**Comment by steve-gregory** [comment]
#295 merged and closes #294.

**Comment by steve-gregory** [comment]
Reopened after rerereading 295, will explore and provide solution for this issue.
